### DISCUSSION TITLE
Add class to html from rich editor

### DISCUSSION THREAD

**Discussion by lironsam** [filamentphp/filament#558] [Help]
Hi,

It may be a silly question but is it possible to add a custom class or attribute to the html that is saved from the rich editor ?

For example
If I write and save _hello world_

It would be save to _"< p >hello world< /p >_ "
And I would like something  like _"< p class="test" >hello world</ p >"_  instead.

Thank you

**Comment by danharrin** [comment]
If you can work out a way to do this with Trix, let me know and I'll work out how you can integrate this into Filament.

**Reply by titantwentyone** [reply]
This is an old query and I have been looking for the same thing with no joy. Trix has two PRs pending from 2020 which seemingly address the issue but these have stagnated.

Quill does do this and with minimal effort. Just thinking that Quill opens up a lot of options for Filament devs to push in their own controls or amend existing ones with custom attributes.

**Reply by dmitrijsmihailovs** [reply]
One option would be to add that class using php. I know that there is handleRecordCreation method in create page component and I guess the something similar in edit page.
Then you could use DOMDocument, load that html and add everything you need.

**Comment by khalidmaquilang** [comment]
too bad.. until now, im searching for answer on this issue

**Reply by holiq** [reply]
already answered? or have any way for do that?
